source/_patterns/00-atoms/references/reference.yaml

```diff
@@ -4,7 +4,10 @@ properties:
   bibId:
     type: string
     minLength: 1
-  titleLink:
+  title:
```

Comment:
This has changed to `title`, should be `titleLink`?

Reply:
Yes this should be titleLink. Not sure why it has been changed back